inc/classes/subscriber/Tools/class-detect-missing-tags-subscriber.php

```diff
@@ -31,24 +31,28 @@ public static function get_subscribed_events() {
 	 * @param string $html HTML content.
 	 */
 	public function maybe_missing_tags( $html ) {
-		Logger::info( 'START Detect_Missing_Tags_Subscriber - maybe_missing_tags ', [ 'maybe_missing_tags' ] );
+		// If there is a redirect the content is empty and can display a false positive notice.
+		if ( strlen( $html ) <= 255 ) {
+			return;
+		}
+		Logger::info( 'START Detect Missing closing tags ( <html>, </body> or wp_footer() )', [ 'URI' => $this->get_raw_request_uri() ] );
```

Comment:
We're not passing the `'maybe_missing_tags'` context anymore?

Reply:
Nope, I taught in this case it's much more relevant to display the exact URL where there are missing tags instead of a plain context. 
What do you think?

Reply:
Why "instead"? We can have both. The context is useful when searching the log.